site/src/xServices/audit/auditXService.ts

```diff
@@ -131,8 +131,12 @@ export const auditMachine = createMachine(
           }).then((data) => data.count),
         ])
 
+        // Filter out any logs that don't have an actual diff to present to the user
+        const filteredLogs = auditLogs.filter(
+          (log) => Object.entries(log.diff).length > 0,
+        )
```

Comment:
@coadler I'm filtering out any logs with an empty diff on the FE to hide duplicate records. Is there a better way to clean these up? Will filtering these out have any unintended side effects, i.e. is it valid to have an audit log without diff content in any use case?

Reply:
It's fairly likely that we'll have audit logs in the future that shouldn't have diffs, do I don't think we should try to hide them.